portal/portal-render-engine-impl/impl/src/webapp/vm/morpheus/includeSiteHierarchy.vm

```diff
@@ -18,8 +18,11 @@
         #end ## END of FOREACH ( $crumb in $portalBreadCrumbs )
         
         #set ($tabCountBread = 0)
-        
-        <div class="Mrphs-hierarchy--parent-sites">
+
+        #set ($hasSubsitesClass      = "#if($portalPwd.size() > 1)phone-hidden#{else}#end")
+        #set ($hasSubsitesPhoneClass = "#if($portalPwd.size() > 1)phone-only#{else}#end")
```

Comment:
Unless I'm going blind, these two conditions look the same, if($portalPwd.size() > 1)

Reply:
Yeah, both conditions are the same, but the value after resolving the ternary is different